src/Graphics/GraphicsDevice.cs

```diff
@@ -852,6 +852,12 @@ public void SetRenderTarget(RenderTargetCube renderTarget, CubeMapFace cubeMapFa
 
 		public void SetRenderTargets(params RenderTargetBinding[] renderTargets)
 		{
+			// D3D11 requires our sampler state to be valid (i.e. not point to any of our new RTs)
+			//  before we call SetRenderTargets. At this point FNA3D does not have a current copy
+            //  of the managed sampler state, so we need to apply our current state now instead of
+            //  before our next Clear or Draw operation.
```

Comment:
```suggestion
			//  of the managed sampler state, so we need to apply our current state now instead of
			//  before our next Clear or Draw operation.
```